Resources/Private/JavaScript/Host/Containers/LeftSideBar/index.js

```diff
@@ -10,17 +10,20 @@ import style from './style.css';
 const {$get} = immutableOperations;
 
 @connect(state => ({
-    isHidden: $get(state, 'ui.leftSideBar.isHidden')
+    isHidden: $get(state, 'ui.leftSideBar.isHidden'),
+    isFullScreen: $get(state, 'ui.fullScreen.isFullScreen')
 }))
 export default class LeftSideBar extends Component {
     static propTypes = {
-        isHidden: PropTypes.bool.isRequired
+        isHidden: PropTypes.bool.isRequired,
+        isFullScreen: PropTypes.bool.isRequired
     };
 
     render() {
         const classNames = mergeClassNames({
             [style.leftSideBar]: true,
-            [style['leftSideBar--isHidden']]: this.props.isHidden
+            [style['leftSideBar--isHidden']]: this.props.isHidden,
```

Comment:
Also, if we leave the sidebars animated, should I also add animation to top and bottom bars? I wasn't sure if the animation is necessary for going fullscreen at all, but it's not hard to add it.